ods_ci/tasks/Tasks/patch_dsc_cr_operatorv2.robot

```diff
@@ -0,0 +1,237 @@
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=modelmeshserving
+#     Component Should Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    workbenches
+
+# Verify Workbench Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=workbenches
+#     Component Should Be Enabled    workbenches
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    datasciencepipelines
+
+# Verify None Installation Profile
+#     [Documentation]
+#     Run Keyword And Continue On Failure    Verify Component Resources    component=none
+#     Component Should Not Be Enabled    datasciencepipelines
+#     Component Should Not Be Enabled    dashboard
+#     Component Should Not Be Enabled    distributedWorkloads
+#     Component Should Not Be Enabled    kserve
+#     Component Should Not Be Enabled    modelmeshserving
+#     Component Should Not Be Enabled    workbenches
+
+
+*** Keywords ***
+Component Should Be Enabled
+    [Arguments]    ${component}    ${dsc_name}=default
+    ${status} =    Verify If Component Is Enabled    ${component}    ${dsc_name}
+    IF    '${status}' != 'true'    Fail
+
+Component Should Not Be Enabled
+    [Arguments]    ${component}    ${dsc_name}=default
+    ${status} =    Verify If Component Is Enabled    ${component}    ${dsc_name}
+    IF    '${status}' != 'false'    Fail
+
+Verify Component Resources
+    [Documentation]    Currently always fails, need a better way to check
+    [Arguments]    ${component}
+    Enable Single Component    ${component}
+    ${filepath} =    Set Variable    ods_ci/tasks/Resources/Files/
+    ${expected} =    Get File    ${filepath}${component}.txt
+    Run    oc get $(oc api-resources --namespaced=true --verbs=list -o name | awk '{printf "%s%s",sep,$0;sep=","}') --ignore-not-found -n opendatahub -o=custom-columns=KIND:.kind,NAME:.metadata.name | sort -k1,1 -k2,2 | grep -v "PackageManifest\\|Event\\|ClusterServiceVersion" > ${filepath}${component}_runtime.txt  # robocop: disable
+    Process Resource List    filename_in=${filepath}${component}_runtime.txt
+    ...    filename_out=${filepath}${component}_processed.txt
+    ${actual} =    Get File    ${filepath}${component}_processed.txt
+    Remove File    ${filepath}${component}_processed.txt
+    Remove File    ${filepath}${component}_runtime.txt
+    Should Be Equal As Strings    ${expected}    ${actual}
+
+Verify If Component Is Enabled
```

Comment:
I'd call it sth like `Is Component Enabled` since it is not actually verifying it